scripts/travis.sh

```diff
@@ -9,5 +9,6 @@ export PATH=/home/travis/.cargo/bin:$PATH
 rustup default nightly
 rustup component add rust-src
 cargo install xargo
+cp src/config.rs.sample src/config.rs
```

Comment:
That seems like a reasonable way to do it in cc3200-rs as well (and addresses your concern about checking the file in accidentally).